src/plugins/console/server/lib/spec_definitions/json/generated/cat.segments.json

```diff
@@ -15,6 +15,8 @@
         "p",
         "pb"
       ],
+      "master_timeout": "",
```

Comment:
Out of scope for this PR, but as a note, we may want to update the generation script to ignore deprecated properties, as a more subtle way of guiding folks to the new usage. 🤔

Reply:
Yeah maybe for our side we can just move this out. We can set up some mechanism but good catch @joshuarrrr!. @kristenTian can we delete line 18 due to it not being inclusive?

Reply:
Will delete all `master_timeout` under `src/plugins/console/server/lib/spec_definitions/json/generated` path